styles.css

```diff
@@ -1,3 +1,793 @@
-* {
-    font-size: 1.5rem;
+body {
+    color: #fff;
+    background: #0D0D11;
+    text-align: center;
+    margin: auto;
+    width: 1920px;
+    height: 3288px;
+}
+
+li a {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    list-style: none;
+    text-decoration: none;
+    color: #fff;
+}
+
+.toggle-button {
+    padding: 0;
+    border: 0;
+    height: 48px;
+    background-color: transparent;
+    cursor: pointer;
+}
+
+.discord-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #fff;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    width: 147px;
+    height: 48px;
+    background-color: #141B22;
+    cursor: pointer;
+}
+
+.connect-button {
+    display: flex;
```

Comment:
`.discord-button` and `.connect-button` share many similar properties, so it makes sense to create a common `.btn` or `.button` class for shared styles like flex, font, padding, etc., and add background color to `.discord-button` and `.connect-button.`

Refactoring your CSS to create a common class for shared styles (.btn or .button) helps reduce code duplication and improves maintainability. By using a base class for shared properties and extending it with specific styles for individual buttons, you make your CSS cleaner and easier to manage. Learn more about DRY (Don’t Repeat Yourself) principles in CSS.